File: Praktikum4/ProgramC/PRAK403-2010817210023-MuhammadAuliaAkbar.c
#include <stdio.h>

int main()
{
    int input1, input2;
    printf("Input: ");
    scanf("%d %d", &input1, &input2);

    if (input1 < input2)
    {
        int batas = input2;
        while (input1 <= batas)
        {
            if (input1 == batas)
            {
                printf("%d %d", input1, input2);
            }
            else
            {
                printf("%d %d - ", input1, input2);
            }
            input1++;
            input2--;
        }
    }
    else if (input1 > input2)
    {
        int batas = input1;
        while (input2 <= batas)
        {
            if (input2 == batas)
            {
                printf("%d %d", input1, input2);
            }
            else
            {
                printf("%d %d - ", input1, input2);
            }
            input1--;
            input2++;
        }
    }
    else
    {
        printf("The two numbers are equal");
    }
    return 0;
}

File: Praktikum4/ProgramPython/PRAK403-2010817210023-MuhammadAuliaAkbar.py
DuaInputan = input("Dua Inputan : ")

input1, input2 = map(int, DuaInputan.split())

if input1 < input2:
    batas = input2
    while input1 <= batas:
        if input1 == batas:
            print(f"{input1} {input2}")
        else:
            print(f"{input1} {input2}", end=" - ")
        input1+= 1
        input2-= 1
    
elif input1 > input2:
    batas = input1
    while input2 <= batas:
        if input2 == batas:
            print(f"{input1} {input2}")
        else:
            print(f"{input1} {input2}", end=" - ")
        input1-= 1
        input2+= 1

else:
    print("The two numbers are equal")